packages/salesforcedx-apex-debugger/package.json

```diff
@@ -42,7 +42,7 @@
     "vscode-uri": "^3.1.0"
   },
   "scripts": {
-    "bundle:debugger": "esbuild ./src/adapter/apexDebug.ts  --bundle --outfile=dist/apexdebug.js --format=cjs --platform=node --minify",
+    "bundle:debugger": "esbuild ./src/adapter/apexDebug.ts  --bundle --outfile=dist/apexdebug.js --format=cjs --platform=node --minify --external:vscode",
```

Comment:
what made the `external` necessary here?  isn't `vscode` just a bunch of Types?

Reply:
`npm run vscode:package` was failing without it, but I'm not sure if it's ok to use it